plugins/inputs/prometheus/prometheus.go

```diff
@@ -38,6 +40,9 @@ var sampleConfig = `
   ## Use bearer token for authorization
   # bearer_token = /path/to/bearer/token
 
+  ## Specify timeout in seconds for slower prometheus clients (default is 3)
+  # response_timeout = 3
```

Comment:
once ResponseTimeout is an `internal.Duration` type, it can be written as `response_timeout = "3s"`